.github/workflows/ci_workflow.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
```

Comment:
In terms of file naming, how about `test.yml` and `deployment.yml`? Since it's already in the workflows directory, having the suffix `_workflow` seems redundant 💭